tests/cypress/tests-e2e/header.cy.js

```diff
@@ -127,6 +127,15 @@ describe('Test functions for the header / search bar', () => {
             checkLangAndTopic('en', 'test-topic-standard')
             cy.readStoreValue('state.layers.activeLayers').should('have.length', 0)
         })
+        // This test is not working because clicking the news button crashes cypress
+        // see https://github.com/cypress-io/cypress/issues/24084 for more information
+        it.skip('should show the news website when clicking on the News button', () => {
+            cy.viewport(1920, 1080)
+            cy.goToMapView()
+            cy.get('[data-cy="header-cms-link"]').click()
+
+            cy.url().should('equal', 'https://www.geo.admin.ch/en/#News')
+        })
```

Comment:
you can move it under the `if (width >= BREAKPOINT_TABLET) {` and not set the viewport size.
It will then be run automatically when we start running tests with bigger viewports